File: cprograms/maximum.c
#include <stdio.h>

int main(void) {
  int Counter, Largest, n;
  printf("Enter among how many numbers you want to find the maximum:");
  scanf("%d", &n);
  int Number[n];
  printf("Enter the first number:");
  scanf("%d", &Number[0]);
  for(Counter = 1; Counter <=n-1 ; Counter++)
  {
    printf("Enter the next number:");
    scanf("%d", &Number[Counter]);
  }
  printf("The maximun of %d numbers is ",n);
  Largest = Number[0];
  for(Counter = 0; Counter <= n-1; Counter++)
  {
    for(Counter = 0; Counter <= n-1; Counter++)
    {
      if(Largest <= Number[Counter])
     {
        Largest = Number[Counter];
     }
   }
  }
  printf("%d.", Largest);
  return 0;
}

File: javascriptprograms/maximum.js
var readln = (typeof readline === 'function') ? (readline) : (function() {
     importPackage(java.io);
     importPackage(java.lang);
     System.out.print("Enter among how many numbers you want to find maximum:");
     var stdin = new BufferedReader(new InputStreamReader(System['in']));

     return function() {
         return String(stdin.readLine());  // Read line, 
     };                                    // force to JavaScript String
 }());
 
 n= parseInt(readln());
var Count = new Array(n);
System.out.print("To find the maximum of " + n + " numbers, Enter the first number:");
Number = parseInt(readln());
Count[0] = Number;
for (Counter = 1; Counter <= n-1; Counter++){
  System.out.print("Enter the next number:");
  Number = parseInt(readln());
  Count[Counter] = Number;
}
Largest = Count[0];
 for(Counter = 0; Counter <= n-1; Counter++)
  {
    for(Counter = 0; Counter <= n-1; Counter++)
    {
      if(Largest <= Count[Counter])
     {
        Largest = Count[Counter];
     }
   }
  }
print("The maximum of " + n + " numbers is " + Largest + ".");